src/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,40 @@
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import { Image as ImageComponent } from '@/components/Image';
+
+const Avatar = ({ src, alt, size, shape, mode }) => {
+  return (
+    <Wrapper size={size} shape={shape}>
+      <ImageComponent src={src} alt={alt} width={size} height={size} type={'profile'} mode={mode} />
```

Comment:
Avatar 하위의 Image 컴포넌트는 프로필 이미지로만 쓰이니까 이렇게 쓰는 건 어떨까요?
```suggestion
      <ImageComponent src={src} alt="프로필 이미지" width={size} height={size} type="profile" mode={mode} />
```